src/sass/mixin/_media-query.scss

```diff
@@ -8,3 +8,9 @@
     @content;
   }
 }
+
+@mixin co-x-media-mobile() {
```

Comment:
We've been trying to refer to things in terms of xs, sm, md, lg instead of specific pixel sizes or device types.
Maybe instead of making this "mobile" specific we add another mixin like the inverse of the above?
i.e. `co-x-media-max($size)`?